mcubes/src/marchingcubes.h

```diff
@@ -74,111 +89,117 @@ void marching_cubes(const vector3& lower, const vector3& upper,
                 if(edges & 0x040)
```

Comment:
vector is very costly here. it's better to rewrite like this one: ```
      std::int32_t indices[12] = {
                        -1, -1, -1, -1,
                        -1, -1, -1, -1,
                        -1, -1, -1, -1,
                    };
```

Reply:
You are absolutely right. I will fix this.

Reply:
It's fixed now. Thank you!